### ISSUE TITLE
problems for pairing, even with the fix

### ISSUE THREAD

**Issue by erwannevonia** [psmoveservice/PSMoveService#703]
so i don't know why, but on my 2 windows ( win 10 last version and win 8 now ) after the first time i paired my move controller, it says to me :
[2020-04-15 02:31:24.444]: AsyncBluetoothPairDeviceRequest - Bluetooth device found matching the given address: a0:ab:51:43:ce:f1
[2020-04-15 02:31:24.451]: AsyncBluetoothPairDeviceRequest - Bluetooth device matching the given address is the expected controller type
[2020-04-15 02:31:24.458]: AsyncBluetoothPairDeviceRequest - Already authenticated.
again and again... but i can't see him on the config tool. also i already the download the psmoveserviceadmin.exe fix, but it doesn't work, and deleting the motion controller in my bluetooth devices ( even the hides ones )
please help, it so frustrating when it doesn't work :(

**Comment by lmnope1** [comment]
I am having a similar issue - I can connect one controller at a time but that is all. I can pair the second one if i turn off the first and they work individually' but I cannot connect them both at the same time. I receive the error given by the OP if I try to re-add them or I get a device not found error if another move controller is already connected

**Comment by Swaiiler48** [comment]
I would love it if someone fixed this. It is my exact problem. I have no idea how to code so I can't do anything.

**Comment by erwannevonia** [comment]
> I would love it if someone fixed this. It is my exact problem. I have no idea how to code so I can't do anything.

Are you on a laptop ? Because I know most players are on a desktop computer, and so they need a bluetooth adapter.

**Comment by lmnope1** [comment]
@erwannevonia Yeah. I bought a USB Bluetooth adapter and disabled my internal one. Connects well now!

**Comment by erwannevonia** [comment]
I was thinking about this solution but never found something on it, so thank you for telling me 👍
